unstoppabledomains/resolution
# CLI

Prototype for the command-line interface for resolution. 

To test locally

1.  checkout the branch
3. npm link (should produce npm install and link the executable with the symlink inside the npm)
4. ping test
`resolution -V`
5. Configure the library with awesome -C <type>:<value> flag. Type can be either infura (for infura project id only) or url (for any specified url). this configuration goes only for ens, CNS for now. Later can extend to configure zilliqa as well. 
5. help menu
`resolution -h`

I need more use cases to improve from this point, any ideas appreciated
